docs/architecture/microservices/implement-resilient-applications/implement-http-call-retries-exponential-backoff-polly.md

```diff
@@ -12,9 +12,9 @@ Polly is a .NET library that provides resilience and transient-fault handling ca
 
 The following steps show how you can use Http retries with Polly integrated into `IHttpClientFactory`, which is explained in the previous section.
 
-**Reference the ASP.NET Core 3.1 packages**
+**Reference the .NET Core 3.1 packages**
 
-`IHttpClientFactory` is available since .NET Core 2.1 however we recommend you to use the latest ASP.NET Core 3.1 packages from NuGet in your project. You typically also need to reference the extension package `Microsoft.Extensions.Http.Polly`.
+`IHttpClientFactory` is available since .NET Core 2.1 however we recommend you to use the latest .NET Core 3.1 packages from NuGet in your project. You typically also need to reference the extension package `Microsoft.Extensions.Http.Polly`.
```

Comment:
`IHttpClientFactory` is available since .NET Core 2.1 however we recommend you to use the latest ASP.NET Core 5 packages from NuGet in your project. You typically also need to reference the extension package `Microsoft.Extensions.Http.Polly`.